test/tasks/demo.test.js

```diff
@@ -95,11 +95,22 @@ describe('Demo task', function() {
 			demo(gulp)
 				.catch(function(err) {
 					setTimeout(function() {
-						expect(err).to.not.be('Couldn\'t find demos config path, checked: demos/src/config.json,demos/src/config.js');
+						expect(err).to.not.be('Couldn\'t find demos config path, checked: demos/src/config.json,demos/src/config.js,origami.json');
+						fs.unlink('demos/src/config.js');
+						done();
 					});
-					fs.unlink('demos/src/config.js');
+				});
+		});
+
+		it('should not fail using origami.json', function(done) {
+			demo(gulp, {
+				demoConfig: 'origami.json'
+			}).catch(function(err) {
+				setTimeout(function() {
+					expect(err).to.not.be('Couldn\'t find demos config path, checked: origami.json');
```

Comment:
So this test passes if the demo task throws an error that doesn't match the string you're checking?